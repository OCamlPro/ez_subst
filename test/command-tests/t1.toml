[template]

src = "input.txt"
dst = "t1.output"
config = "c0.toml"


