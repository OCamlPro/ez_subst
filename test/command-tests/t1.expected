Hello World
$x $y
$[x] $[y]
${x} ${y}
$(x) $(y)
